docs/README.md

```diff
@@ -1,10 +1,18 @@
-# `kind` Documentation
+# Welcome to `kind`'s Documentation
 
 
-For usage documentation see the [user guide][user guide].
+## Using `kind`
+If you want to get started with `kind`, checkout our [user guide][user guide].
+Here you will find all details to get you started along with some more advanced
+usage cases.
 
-For the primary design start with [design][design].
+If you want to see the broader picture and understand the why and how of `kind`
+then go and checkout our [design docs][design].
+
+For more specific information about how things are implemented and how `kind`
```

Comment:
Probably instead we want to say something about "If you're interested in modifying or contributing to `kind` ..." since the design docs should cover how things are implemented for the most part (eventually, if they don't now).

Otherwise, I love the changes to this file! 😄